src/vcsparser.core/ChangesetProcessor.cs

```diff
@@ -47,11 +51,14 @@ public void ProcessChangeset(IChangeset changeset)
             if (!dict.ContainsKey(changeset.ChangesetTimestamp.Date))
                 dict.Add(changeset.ChangesetTimestamp.Date, new Dictionary<string, DailyCodeChurn>());
 
-            bool containsBugs = CheckAndIncrementIfChangesetContainsBug(changeset);            
+            bool containsBugs = CheckAndIncrementIfChangesetContainsBug(changeset);
 
             foreach (var c in changeset.ChangesetFileChanges)
             {
                 ProcessFileChange(changeset, containsBugs, c);
+
+                if (workItemCache.ContainsKey(changeset.ChangesetIdentifier.ToString()))
+                    ProcessBugDatabaseFileChange(changeset, containsBugs, c);

```

Comment:
There's no dependency between containsBug and bug database. In this current logic you will only count this changeset as bugs if it matches the regex AND exists in the bug database. That's not the requirement. The idea is to do independent metrics. It may match the regex and not exist in the bug database and vice-versa.

Reply:
Removed the dependency of `containsBugs` in 436638d125bb40b11223df403d8dd9b234d55800